acestep/ui/gradio/events/wiring/generation_text_format_wiring.py

```
@@ -53,6 +53,44 @@ def register_generation_text_format_handlers(
+        fn=lambda caption, lyrics: (caption or "", lyrics or ""),
+        inputs=[
+            generation_section["captions"],
+            generation_section["lyrics"],
+        ],
+        outputs=[
+            generation_section["flow_edit_source_caption"],
+            generation_section["flow_edit_source_lyrics"],
+        ],
+    )
+
+    # ========== Retake × Think interaction warning ==========
```

Ruff (0.15.12)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)